ckanext/xloader/jobs.py

```diff
@@ -111,7 +111,7 @@ def get_default_queue_name(package_id=None):
         return DEFAULT_QUEUE_NAME
     if not package_id:
         return DEFAULT_QUEUE_NAMES[0]
-    return DEFAULT_QUEUE_NAMES[hash(package_id) % len(DEFAULT_QUEUE_NAMES)]
+    return DEFAULT_QUEUE_NAMES[ord(package_id[0]) % len(DEFAULT_QUEUE_NAMES)]
```

Comment:
please add comment here what ord is doing on picking queue. or into the function description.